src/main/features/response/views/counter-claim.njk

```diff
@@ -1,47 +1,39 @@
 {% extends "layout.njk" %}
 {% from "form.njk" import radioGroup, saveAndContinueButton, errorSummary %}
 {% from "support.njk" import supportWidget %}
+{% from "feesTable.njk" import feesTable %}
 
 {% set title = t('Money claim') %}
 
-{% set heading = t('Next steps') %}
+{% set heading = t('How to complete and send your response') %}
 
 {% block content %}
   <div class="grid-row">
     <div class="column-two-thirds">
       <div class="panel-indent">
         <div class="panel panel-border-wide">
-          <p>{{ t('Complete and email the defence and counterclaim form by %s',
-            { postProcess: 'sprintf', sprintf: [claim.responseDeadline | date]}) }}</p>
-          <p>{{ t('To make a counter claim you will have to pay a fee. Court staff can tell you how much you have to pay.') }}</p>
+          <p>{{ t('Complete and email the defence and counterclaim form by') }} <span class="bold-small">{{ claim.responseDeadline | date }}</span></p>
         </div>
       </div>
 
+      <p>{{ t('You’ll need to pay a court fee if you make a counterclaim.') }}</p>
+
+      {{ feesTable('View claim fees', rows, hidden = true ) }}
+
       <ul class="list list-number">
         <li>
-          {{ t('Complete <a href="%s" target="_blank">defence and counterclaim form N9B</a> <span class="font-xsmall">(opens in a new window)</span> entering these details:',
-          { postProcess: 'sprintf', sprintf: [asset_paths["pdf"] + '/OCON9B.pdf']}) | safe }}
+          {{ t('Complete <a href="{{ link }}" download="download" target="_blank">defence and counterclaim form N9B</a> entering these details:',
+          { link : [asset_paths["pdf"] + '/OCON9B.pdf']}) | safe }}
           <ul class="list list-bullet">
             <li>{{ t('Claim No.') }} {{ claim.claimNumber }}</li>
             <li>{{ t('Claimant') }} {{ claim.claimData.claimant.name }}</li>
             <li>{{ t('Defendant') }} {{ response.defendantDetails.partyDetails.name }}</li>
           </ul>
         </li>
-        <li>{{ t('Download and attach the completed form to an email') }}</li>
-        <li>{{ t('Send the email with the attached form to') }} <a href="mailto:{{ supportEmailAddress }}?Subject={{ claim.claimNumber }}">{{ supportEmailAddress }}</a></li>
+        <li>{{ t('Save the completed form to your device.') }}</li>
+        <li>{{ t('Email the form to') }} <a href="mailto:{{ supportEmailAddress }}?Subject={{ claim.claimNumber }}">{{ supportEmailAddress }}</a></li>
```

Comment:
Similar case here, whole `<li>` content should be translated.

Reply:
done